src/style.css

```diff
@@ -0,0 +1,75 @@
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.flex_container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+
+  max-width: 1199px;
```

Comment:
Why you add max_width?
check list point - 5
Block should strech

![image](https://user-images.githubusercontent.com/63306833/150757653-18a9d415-9531-4a94-a518-0a6db7474b7c.png)